core/src/fields.rs

```diff
@@ -725,23 +725,23 @@ impl MongoStatement for MongoFields {
                 match get_meta_data()?.column_size {
                     // If the driver cannot determine the column for a variable type, it returns
                     // SQL_NO_TOTAL.
-                    None => definitions::SQL_NO_TOTAL as i32,
-                    Some(col_size) => col_size as i32,
+                    None => i32::try_from(definitions::SQL_NO_TOTAL).expect("Cannot convert i32"),
```

Comment:
I agree with the idea of defining a SQL_NO_TOTAL_i32 version of this. We should have one version of each constant per type we ever send it through.

Reply:
Agreed!
It would avoid the awkward/useless `.expect("Cannot convert i32")` since wek know that the value it will be Ok.

Reply:
I've updated these to be Integer (i32). Turns out there was no reason to have them as isize at all, the reason we were always casting down to i32 is because all functions that take a text_length expect it as Integer.